scripts/bootstrap.sh

```diff
@@ -180,7 +180,7 @@ get_user_details() {
   local id="$1"
 
   # shellcheck disable=SC2016
-  local query='{"query":"query GetUserDetails($id: String!) {user(userId: $id) {id email displayName firstName lastName creationDate uuid groups {id displayName}}}","operationName":"GetUserDetails"}'
+  local query='{"query":"query GetUserDetails($id: String!) {user(userId: $id) {id email displayName firstName lastName creationDate uuid groups {id displayName} attributes {name value}}}","operationName":"GetUserDetails"}'
```

Comment:
Do you think you can format it better with a heredoc?

Reply:
Sure! I've added a heredoc to make this more readable but I don't think I can split the graphql query into multiple lines because it's defined as a string in a JSON object?

Reply:
Ah, damn, JSON is indeed pretty limiting. Feel free to revert to the previous version without the extra cat, it seems simpler

Reply:
Agreed. Reverted to original change.